nautobot/docs/user-guide/feature-guides/contact-and-team.md

```diff
@@ -0,0 +1,52 @@
+# Contacts and Teams
+
+## How to Assign Contacts and Teams to Nautobot Objects
+
+In order to assign contacts/teams to a particular object, we first need to navigate to the detail view of the object. We will use a circuit in this example.
+
+![Circuit](./images/contact-and-team/circuit.png)
+
+At the top right corner, you should see a group of buttons and one of them should say **Add Contact**.
+
+![Circuit Add Contact Button](./images/contact-and-team/circuit_button.png)
+
+Click on the **Add Contact** button and it will redirect you to a page with forms on three separate tabs.
+
+![Contact Form](./images/contact-and-team/contact_forms.png)
+
+1. The first tab contains a form that allows the user to create a **new** contact and associate it with the object.
+2. The second tab contains a form that allows the user to create a **new** team and associate it with the object.
+3. The third tab contains a form that allows the user to select an **existing** contact or team and associate it with the object.
+
+![Contact Form Tabs](./images/contact-and-team/contact_form_tabs.png)
+
+### Assign a New Contact/Team
+
+Which form to use depends on the use case but in this example, we will use the first form which creates a **new** contact instance and associates it with the object.
+
+1. We first fill out the contact information.
+![Contact Info](./images/contact-and-team/new_contact_info.png)
+2. Then we fill out the information needed for the assignment of this contact to the object.
+![Contact Association Info](./images/contact-and-team/new_contact_association_info.png)
```

Comment:
Would be nice to use somewhat more realistic data here. Status `Active` is probably fine to use, and I'd experimented with roles like `L1 Support` and `Manager` for contact associations.